public/index.html

```diff
@@ -18,107 +22,265 @@
+        <img class="solution__section__img5-bg" src="assets/images/dollar-bills-bg.jpg">
+        <img class="solution__section__img5" src="assets/images/dollar-bills.png">
+        <h3 class="solution__section__hdr">She finds you deals.</h3>
+        <p class="solution__section__content">Because Florence knows your regimen, she can find you ways to reduce the cost of care.</p>
+      </div>
+      <div class="solution__section__right">
+        <ol class="bubble">
+          <li class="bubble__florence">BTW, Walmart is now offering $4 generics. Choosing warfarin, the generic form of Coumadin, would save you $15. I'll remind you when when you're up for a refill.</li>
+          <li class="bubble__patient">Thanks</li>
+          <li class="bubble__florence--tail">Certainly</li>
+        </ol>
+        <img class="solution__section__right__avatar--5" src="assets/images/florence-avatar.svg">
+      </div>
+    </section>
+
+    <section role="region" class="solution__section--6">
+      <div class="solution__section__left">
+        <h3 class="solution__section__hdr">She cheers you on.</h3>
+        <p class="solution__section__content">Florence can send you occasional notes of encouragement to keep you motivated.</p>
+        <p class="solution__section__content">For patients with asymptomatic conditions, this offers a sense of progress.</p>
+        <img class="solution__section__img6-bg" src="assets/images/section6-illo-bg.jpg">
+        <img class="solution__section__img6" src="assets/images/section6-illo.svg">
+      </div>
+      <div class="solution__section__right">
+        <ol class="bubble">
+          <li class="bubble__florence">Hey Mike! Nice job, your cholesterol is down 20%.</li>
+          <li class="bubble__florence--tail">Treat yourself to something nice! <img class="emoji" src="assets/images/happy.svg"></li>
+        </ol>
+        <img class="solution__section__right__avatar--6" src="assets/images/florence-avatar.svg">
+      </div>
+    </section>
+
+    <section role="region" class="solution__section--7">
+      <div class="solution__section__left">
+        <h3 class="solution__section__hdr">She keeps you honest.</h3>
+        <p class="solution__section__content">If Florence notices you’re not staying on top of your plan, she can message family members or others in your support network.</p>
+        <img class="solution__section__img7-bg" src="assets/images/section7-illo-bg.jpg">
+        <img class="solution__section__img7" src="assets/images/section7-illo.svg">
+      </div>
+      <div class="solution__section__right">
+        <ol class="bubble">
+          <li class="bubble__florence--tail">Hi Kim, this is an automated message from FlorenceApp.io Your brother Mike hasn't taken his meds in three days. You might want to give him a call. <a class="bubble__number" href="">(919) 232-0232</a></li>
+        </ol>
+        <img class="solution__section__right__avatar--7" src="assets/images/florence-avatar.svg">
```

Comment:
Hmmm... Random thought. I'm wondering if maaaybe these conversations could be grouped in a slightly more semantic `<figure>` since it's an example...  That one is debatable, so I'll let you decide :)